### PR TITLE
add more articles and videos

### PR DESCRIPTION
**Pull request by anwarmamat**
*Description of changes:*

Adds new blogs and videos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,15 @@ A curated list of [awesome](https://github.com/sindresorhus/awesome) Cedar relat
   - [How Cedar was built with verification-guided development](https://www.amazon.science/blog/how-we-built-cedar-with-automated-reasoning-and-differential-testing)
   - [Cedar: A new policy language](https://onecloudplease.com/blog/cedar-a-new-policy-language)
   - [Scaling Authorization with Cedar and OPAL](https://www.permit.io/blog/scaling-authorization-with-cedar-and-opal)
+  - [Using Open Source Cedar to Write and Enforce Custom Authorization Policies](https://aws.amazon.com/blogs/opensource/using-open-source-cedar-to-write-and-enforce-custom-authorization-policies/)
+  - [https://www.amazon.science/blog/how-we-built-cedar-with-automated-reasoning-and-differential-testing](https://www.amazon.science/blog/how-we-built-cedar-with-automated-reasoning-and-differential-testing)
```
Comment (khieta) [inline README.md:30]: Line 30 is a duplicate of line 26. Also note that the title on line 26 is incorrect; it should be "How we built Cedar with automated reasoning and differential testing"

File: README.md [hunk 2]
```diff
@@ -26,10 +26,15 @@ A curated list of [awesome](https://github.com/sindresorhus/awesome) Cedar relat
...
+  - [The Cedar Programming Language: Authorization Simplified](https://thenewstack.io/the-cedar-programming-language-authorization-simplified/)
+  - [Verification-Guided Development of the Cedar Authorization Language](https://cps-vo.org/node/94470)
```
Comment (khieta) [inline README.md:32]: The HCSS page isn't an article, it's an abstract w/ slides. You could include it in a "talks" section, or just leave it out.

File: README.md [hunk 3]
```diff
@@ -26,10 +26,15 @@ A curated list of [awesome](https://github.com/sindresorhus/awesome) Cedar relat
...
 
 ## Videos
   - [Use policies to manage permissions w/Amazon Verified Permissions (AWS re:Invent 2022)](https://www.youtube.com/watch?v=k6pPcnLuOXY)
-
+  - [Demo of the Cedar Programming Language - The Open Source Language from AWS](https://www.youtube.com/watch?v=PzmDYyyA5xM)
+- [The Cedar Language and Policy Based Authorization with Emina Torlak](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587/55f50479)
```
Comment (khieta) [inline README.md:37]: Indentation typo

**Review by khieta — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anwarmamat) [reply README.md:30]: fixed

File: README.md (hunk 3 above)
Comment (anwarmamat) [reply README.md:37]: fixed

**Review by khieta — APPROVED** [review]

**Review by andrewmwells-amazon — APPROVED** [review]

**Review by john-h-kastner-aws — APPROVED** [review]

**Pull request merged**